js/index.js

```diff
@@ -28,3 +28,46 @@ for (let listItem of skills) {
     
     //console.log(listItem);//
 };
+
+//lesson 4-3
+messageForm = document.getElementsByName('leave_message')[0];
+//console.log(messageForm);
+
+//create submit button for message form
+messageForm.addEventListener('submit', myFunction => {
+    //console.log('event listener working');
+    myFunction.preventDefault();
+    const name = myFunction.target.name.value;
+    //console.log(name);
+    const email =  myFunction.target.email.value;
+    //console.log(email);
+    const message = myFunction.target.message.value;
+    //console.log(message);
+
+    //console.log('3 variables successfully created');
+
+    //Display messages in list
+    const messageSection = document.getElementById('message');
```

Comment:
hi, could you explain what you are trying to get with getElementById since I do not find the definition by the line message 50 thank you =)